build-system/server.js

```diff
@@ -551,6 +555,18 @@ app.get('/dist/rtv/99*/*.js', function(req, res, next) {
   }).catch(next);
 });
 
+app.get(['/dist/rtv/*/v0/amp-ad-0.1.js', '/dist/rtv/*/v0/amp-video-0.1.js'], function(req, res, next) {
```

Comment:
you can do `'/dist/rtv/*/v0/:filename'`
then you can retrieve filename from [req.params.filename](http://expressjs.com/en/4x/api.html#req.params)